examples/voip/Client.cpp

```diff
@@ -67,12 +67,12 @@ class NetworkRecorder : public sf::SoundRecorder
     /// \see SoundRecorder::onProcessSamples
     ///
     ////////////////////////////////////////////////////////////
-    bool onProcessSamples(const sf::Int16* samples, std::size_t sampleCount) override
+    bool onProcessSamples(std::span<const sf::Int16> samples) override
```

Comment:
Is it convention to pass spans by value when you just need to read from the span? I haven't worked with them before.

Reply:
Passing a span by value is similar to passing a pointer and a size (both by value).
This what [C++ Core Guidelines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f24-use-a-spant-or-a-span_pt-to-designate-a-half-open-sequence) recommends.